rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/RestControllerHelper.java

```diff
@@ -941,4 +941,22 @@ public void addEmbeddedModerationRequest(HalResource halResource, ModerationRequ
         halResource.addEmbeddedResource(isSingleRequest ? "sw360:moderationRequest" : "sw360:moderationRequests",
                 halModerationRequest);
     }
+
+    public void addEmbeddedModifiedByToRelease(HalResource halResource, Component sw360Component) {
+        if (sw360Component.getModifiedBy() != null) {
+            User componentModify = getUserByEmail(sw360Component.getModifiedBy());
+            if (null != componentModify)
+                addEmbeddedUser(halResource, componentModify, "modifiedBy");
+        }
+    }
+
+    public void addEmbeddedComponentOwnerToRelease(HalResource halResource, Component sw360Component) {
+        if (sw360Component.getComponentOwner() != null) {
+            User componentOwner = getUserByEmail(sw360Component.getComponentOwner());
+            if (null != componentOwner) {
+                addEmbeddedUser(halResource, componentOwner, "componentOwner");
+                sw360Component.setComponentOwner(null);
```

Comment:
What's the point of line number 958 when you already checking null condition on line 956?

Reply:
The Data ComponentOwner has been added to the Embedded section so it doesn't need to be set in the Component Response.